specification/network/resource-manager/Microsoft.Network/stable/2023-05-01/virtualNetwork.json

```diff
@@ -1631,6 +1631,10 @@
             "$ref": "./applicationGateway.json#/definitions/ApplicationGatewayIPConfiguration"
           },
           "description": "Application gateway IP configurations of virtual network resource."
+        },
+        "defaultOutboundAccess": {
+          "type": "boolean",
```

Comment:
>boolean [](http://example.com/codeflow?start=19&length=7)

consider using enum

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md)

Reply:
Updated Property type